modules/installation-obtaining-installer.adoc

```diff
@@ -23,23 +23,24 @@ computer.
 have a Red Hat account, log in with your credentials. If you do not, create an
 account.
 
-. Download the installation program and place the file in the directory where
-you will store the installation configuration files.
+. Download the installation program for your operating system and place the file
+in the directory where you will store the installation configuration files.
 +
 [IMPORTANT]
 ====
 The installation program creates several files on the computer that you use to
-install your cluster. While you can delete the installation program after you
-install a cluster, you must keep the files that the installation program
-creates.
+install your cluster. You must keep both the installation program and the files
```

Comment:
Well, you can always re-download the installer.  But yeah, keeping it is probably easier.

Reply:
If we're sure that people can find the right versions to successfully uninstall or that any version of the installer will work, I'm happy to change this.